data/json/recipes/recipe_food.json

```diff
@@ -9106,6 +9106,29 @@
   ]  
 },{
   "type" : "recipe",
+   "result" : "hotdogs_campfire",
+   "category" : "CC_FOOD",
+  "subcategory" : "CSC_FOOD_MEAT",
+   "skill_used" : "cooking",
+   "difficulty" : 0,
+   "time" : 1000,
+   "reversible" : false,
+   "autolearn" : true,
+   "batch_time_factors" : [ 67, 5 ],
+   "qualities" : [
+     { "id" : "COOK","level" : 1,"amount" : 1 }
+  ], "tools" : [
+    [ [ "hotplate", 10], [ "char_smoker", 10], [ "toolset", 10], [ "fire", -1] ],
+    [ [ "spear_wood", -1 ], [ "spear_steel", -1 ], [ "pointy_stick", -1 ], [ "skewer", -1 ], [ "skewer_bone", -1 ],
+      [ "javelin", -1 ], [ "javelin_iron", -1 ], [ "spear_knife", -1 ], [ "tongs", -1 ] ]
```

Comment:
I'd say yes, though in different PR.

Gimmicks like that aren't worth hardcoding individual tools. If "hanging stuff over fire" is needed, it could become its own flag. Though even then, skewers are pretty much free.